tests/crm/test_connectors_onboarding.py

```
@@ -0,0 +1,535 @@
+
+class _FakeInstallationAccessor:
+    """Stands in for db/accessors/installation."""
+
+    def __init__(self, upsert_returns: Optional[InstallationRead] = None):
+        """Test double."""
+        self.upsert_returns = upsert_returns
+        self.written: Optional[Dict[str, Any]] = None
+
+    async def upsert_installation(self, txn, *args):
+        """Test double: record the write by field name, return the seeded row."""
+        self.written = dict(zip(_UPSERT_FIELDS, args))
```

Ruff (0.16.3)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)